frontend/src/pages/ApiEntreprise/demarches.json

```diff
@@ -305,5 +305,55 @@
         "extrait_court_inpi": true
       }
     }
+  },
+  "atexo": {
+    "label": "Éditeur Atexo - Dématérialisation des marchés publics",
+    "about": "https://entreprise.api.gouv.fr/use_cases/marches_publics/",
+    "team_members": {
+      "contact_metier": {
+        "email": "produits@atexo.com",
+        "phone_number": "0175440934"
+      },
+      "responsable_technique": {
+        "email": "regis.menet@atexo.com",
```

Comment:
this trigger a warning:

![image](https://user-images.githubusercontent.com/4190728/153049336-a58561a2-448f-4216-8e63-469ebf2e3765.png)

Reply:
ils n'ont pas réagit à mes mails, j'ai donc mis celui qu'ils ont mis dans leur demande existante

Reply:
C'est pas mon rôle de te bloquer la dessus non. Mais c'est juste pour t'avertir, ça peut être contre productif si les fournisseurs de services pense avoir fait une erreur en utilisant ce cas d'usage.